lib/queryBuilder/graphInserter/DependencyGraph.js

```diff
@@ -126,12 +126,14 @@ class DependencyGraph {
         }
       }
 
-      if (Array.isArray(relModels)) {
-        for (let i = 0, l = relModels.length; i < l; ++i) {
-          this.buildForModel(rel.relatedModelClass, relModels[i], node, rel, nextAllowed);
+      dataPath = `${dataPath}/${rel.name}`;
+      const relModelClass = rel.relatedModelClass;
+      if (Array.isArray(relData)) {
+        for (let i = 0, l = relData.length; i < l; ++i) {
+          this.buildForModel(relModelClass, relData[i], node, rel, nextAllowed, `${dataPath}/${i}`);
         }
-      } else if (relModels) {
-        this.buildForModel(rel.relatedModelClass, relModels, node, rel, nextAllowed);
```

Comment:
@koskimas I've renamed `relModels` to the shorter `relData` to avoid Prettier moving every argument to a new line here... It's also the reason why I'm putting `rel.relatedModelClass` into its own variable.

Reply:
There is no need to fight `prettier`. The whole point of using `prettier` is to accept its style choices and just go with them.

Reply:
No need to change this back though.. I just meant in the future, no need to do stuff like this

Reply:
Oh I see. I can change it if you like, no problem